### DISCUSSION TITLE
使用stash按照要求进行了配置，但无法使用weatherkit，stash中日志如下

### DISCUSSION THREAD

**Discussion by shuke-flies** [NSRingo/.github#18] [Q&A 问答] [answered]
![微信图片_20241117222215](https://github.com/user-attachments/assets/8d9975b2-f665-4b23-b9d3-5fe9a318ae69)
![微信图片_20241117222226](https://github.com/user-attachments/assets/9d150967-62f9-453a-bff4-4bcff5dd08e6)
![微信图片_20241117222230](https://github.com/user-attachments/assets/75958413-10de-4a3b-a76b-ce9227e0b6c1)
日志如下：
[INFO] [23:45:55] [INIT] use config file: /private/var/mobile/Containers/Shared/AppGroup/33C18A5C-127A-41B7-A974-1D3F8AD6B5E1/stash/config.yaml
[INFO] [23:45:55] [DB] 198 fake ip records loaded from disk
[INFO] [23:45:55] [CORE] stash core started, used memory: 8.7M
[WARN] [23:45:56] [RES] pull https://github.com/sub-store-org/Sub-Store/releases/latest/download/sub-store-0.min.js error: Get "https://github.com/sub-store-org/Sub-Store/releases/latest/download/sub-store-0.min.js": EOF
[WARN] [23:45:57] [RES] pull https://github.com/sub-store-org/Sub-Store/releases/latest/download/sub-store-1.min.js error: Get "https://github.com/sub-store-org/Sub-Store/releases/latest/download/sub-store-1.min.js": EOF
[WARN] [23:45:58] [RES] pull https://raw.githubusercontent.com/Script-Hub-Org/Script-Hub/main/script-hub.js error: Get "https://raw.githubusercontent.com/Script-Hub-Org/Script-Hub/main/script-hub.js": EOF
[WARN] [23:45:59] [RES] pull https://github.com/NSRingo/GeoServices/releases/download/v4.0.9/request.js error: Get "https://github.com/NSRingo/GeoServices/releases/download/v4.0.9/request.js": EOF
[WARN] [23:46:00] [RES] pull https://raw.githubusercontent.com/Script-Hub-Org/Script-Hub/main/rule-parser.js error: Get "https://raw.githubusercontent.com/Script-Hub-Org/Script-Hub/main/rule-parser.js": EOF
[WARN] [23:46:01] [RES] pull https://raw.githubusercontent.com/Script-Hub-Org/Script-Hub/main/script-converter.js error: Get "https://raw.githubusercontent.com/Script-Hub-Org/Script-Hub/main/script-converter.js": EOF
[INFO] [23:46:23] [API] system report network is unreachable
[INFO] [23:46:24] [API] system report network is unreachable
[INFO] [23:46:40] [API] system report network is unreachable
[WARN] [23:46:41] [DPC] <TCP> connect radio.itunes.apple.com:443 via DIRECT match DOMAIN-SUFFIX/itunes.apple.com error: resolve ip error: couldn't find ip
[WARN] [23:46:41] [DPC] <TCP> connect amp-api.shazam.apple.com:443 via DIRECT match DOMAIN-SUFFIX/apple.com error: resolve ip error: couldn't find ip
[INFO] [23:46:42] [API] system report network type changed to wifi, ipv4=true, ipv6=true, hotspot=false
[WARN] [23:46:43] [DPC] <TCP> connect 35-courier.push.apple.com:5223 via DIRECT match DOMAIN-SUFFIX/apple.com error: resolve ip error: couldn't find ip
[WARN] [23:46:43] [DPC] <TCP> connect radio-activity.itunes.apple.com:443 via DIRECT match DOMAIN-SUFFIX/itunes.apple.com error: resolve ip error: couldn't find ip
[WARN] [23:46:44] [DPC] <TCP> connect aod.itunes.apple.com:443 via DIRECT match DOMAIN-SUFFIX/itunes.apple.com error: resolve ip error: couldn't find ip

**Comment by VirgilClyne** [comment, marked as answer]
#15
